speed-dreams/source-2.2.3/src/interfaces/track.h

```diff
@@ -670,6 +675,10 @@ typedef struct Track
     int           nseg;         /**< Number of segments */
     int           version;      /**< Version of the track type */
     tdble         length;       /**< main track length */
+    /// SIMULATED DRIVING ASSISTANCE: added the Time property
+    tdble         Time;         /**< main track time (estimated) */  
+    // SIMULATED DRIVING ASSISTANCE: added the SpeedLimit property
+    const char    *SpeedLimit;     /**< speed limit of the track */
```

Comment:
Use tdble (or just float)
```suggestion
   tdble SpeedLimit;     /**< speed limit of the track */
```